### PR TITLE
new searcher function

### PR DESCRIPTION
**Pull request by plankan39**

### CODE REVIEW

File: Search.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import os
+
+
+def search(args):
+    if args[0] != "search":
+        return ["wrong format"]
+
+    searchFile = args[-1]
+    if not os.path.isfile(searchFile):
+        return ["could not find file: " + searchFile]
```
Comment (Andreasjankarlsson) [inline Search.py:17]: Hmm, clever solution!

File: Search.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+def main():
+    args = [""]
+    while args[0] != "q":
+        args = input("enter command('q' to quit): ").split(" ")
+        res = search(args)
+        for line in res:    
+            print(line)
+
+if __name__ == '__main__':
```
Comment (Andreasjankarlsson) [inline Search.py]:
```suggestion
outsideMethod(); // please, dont use a while loop in main <3
```

**Review by Andreasjankarlsson — CHANGES_REQUESTED** [review]
do more!

File: Search.py (hunk 2 above)
Comment (plankan39) [reply Search.py]: fixed with new commit

**Review by Andreasjankarlsson — APPROVED** [review]
Looks fine!

**Pull request merged**